beacon/dkg_runner.go

```diff
@@ -172,6 +172,17 @@ func (dkgRunner *DKGRunner) OnBlock(blockHeight int64, entropy types.ThresholdSi
 	dkgRunner.mtx.Unlock()
 }
 
+//GetNextAeonStart returns the next for entropy generation
+func (dkgRunner *DKGRunner) NextAeonStart(height int64) int64 {
+	dkgRunner.mtx.Lock()
+	defer dkgRunner.mtx.Unlock()
+
+	if height != dkgRunner.height+1 {
+		panic(fmt.Sprintf("consensus state requested next aeon start for unexpected height %v. Expected %v", height, dkgRunner.height+1))
+	}
+	return dkgRunner.aeonStart
+}
+
```

Comment:
Function for passing the NextAeonStart to consensus state for inclusion into BlockHeader. This allows app to trigger validator changeovers required before the start of the aeon.